docs/content/preview/quick-start/docker.md

```diff
@@ -88,8 +88,7 @@ To create a 1-node cluster with a replication factor (RF) of 1, run the followin
 
 ```sh
 docker run -d --name yugabyte -p7000:7000 -p9000:9000 -p15433:15433 -p5433:5433 -p9042:9042 \
- yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \
- --daemon=false
+ yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} bin/yugabyted start \ --background=false
```

Comment:
I seems that a new line is missing. Should be:
```
bin/yugabyted start 
\ --background=false
```

Reply:
Thanks @FranckPachot - addressed this in another [PR](https://github.com/yugabyte/yugabyte-db/pull/20109).